sktime/forecasting/model_selection/_split.py

```diff
@@ -367,10 +363,10 @@ def split(self, y: ACCEPTED_Y_TYPES) -> SPLIT_GENERATOR_TYPE:
         """
         y_index = self._coerce_to_index(y)
 
-        if not isinstance(y_index, pd.MultiIndex):
```

Comment:
hm, this one I would leave like it is, based on "default case first"

not a strong opinion though

Reply:
I don't have a strong opinion on this either, so I restored the previous code.